app/controllers/admin/data_update_scripts_controller.rb

```diff
@@ -3,7 +3,16 @@ class DataUpdateScriptsController < Admin::ApplicationController
     layout "admin"
 
     def index
-      @data_update_scripts = DataUpdateScript.all
+      @data_update_scripts = DataUpdateScript.order(run_at: :desc)
     end
+
+    def run_data_update_scripts
+
+      data_update_script_id = params[:id]
+      response = DataUpdateScript.force_run(data_update_script_id)
+
+      render json: { response: response }
+    end
```

Comment:
This should obviously take into account the possibility that the id doesn't exist when `force_run` raises a 404